src/layouts/Login/LoginPage.tsx

```diff
@@ -0,0 +1,221 @@
+import {
+  Divider,
+  Flex,
+  GridItem,
+  GridProps,
+  Text,
+  Grid,
+  HStack,
+  VStack,
+  Tabs,
+  TabList,
+  TabPanels,
+  TabPanel,
+} from "@chakra-ui/react"
+import {
+  Button,
+  GovtMasthead,
+  Tab,
+  InlineMessage,
+  Link,
+} from "@opengovsg/design-system-react"
+import { useEffect, useState, PropsWithChildren } from "react"
+
+import { useLogin, useVerifyOtp } from "hooks/loginHooks"
+
+import { useSuccessToast } from "utils/toasts"
+
+import { IsomerLogo, LoginImage, OGPLogo } from "assets"
+
+import { LoginForm, LoginProps, OtpForm, OtpProps } from "./components"
+
+const LOGIN_GRID_LAYOUT: Pick<
+  GridProps,
+  | "gridTemplateAreas"
+  | "gridTemplateColumns"
+  | "gridTemplateRows"
+  | "height"
+  | "width"
+> = {
+  gridTemplateAreas: `"image content" 
+                      "credits links"`,
+  gridTemplateColumns: "2fr 4fr",
+  gridTemplateRows: "1fr 5rem",
+  height: "100%",
+  width: "100%",
+}
+
+interface FooterLinkProps {
+  link: string
+}
+
+const FooterLink = ({
+  link,
+  children,
+}: PropsWithChildren<FooterLinkProps>): JSX.Element => (
+  <Link
+    href={link}
+    rel="noopener noreferrer"
+    target="_blank"
+    textDecorationLine="none"
+    color="text.link.dark"
+    _hover={{
+      color: "text.link.dark",
+      textDecorationLine: "underline",
+    }}
+  >
+    {children}
+  </Link>
+)
+
+const LoginContent = (): JSX.Element => {
+  const { mutateAsync: sendLoginOtp, error: loginError } = useLogin()
+
+  const { mutateAsync: verifyLoginOtp, error: verifyError } = useVerifyOtp()
+
+  const successToast = useSuccessToast()
+  const [email, setEmail] = useState<string>("")
+  const [loginErrorMessage, setLoginErrorMessage] = useState("")
+  const [verifyErrorMessage, setVerifyErrorMessage] = useState("")
+
+  const handleSendOtp = async ({ email: emailInput }: LoginProps) => {
+    const trimmedEmail = emailInput.trim()
+    await sendLoginOtp({ email: trimmedEmail })
```

Comment:
Just checking - the backend does send a 200 status code on success, should the return type not be void?

Reply:
oops, my bad above wrt example as there's a `status` prop that we can use to check whether it's succesful or not. what should happen here is either: 

1. add a `useEffect` that listens on the `isSuccess` prop 
2. add the error handling here

i'm more in favour of the former (`useEffect`) because the _side-effect_ is the toast, and i'm inclined to separate it from function call.

Reply:
We're actually getting the errors via `isError` from the `useMutation` hooks, so we're already checking for any errors!

Reply:
Actually as is the error toast only shows up if the call is successful - do you think it's better to put in the conditional for clarity?

Reply:
discussion offline: no need for this here, as non 2xx errors will be caught by axios and thrown as errors; this means that the `successToast` will only be displayed for 2xx status codes, which implies success!